esphome/onju-voice-microwakeword.yaml

```diff
@@ -408,11 +540,6 @@ script:
               - switch.is_on: use_wake_word
           then:
             - micro_wake_word.start
-            - if:
```

Comment:
If I am understanding you correctly, you are suggesting we stop the media player if we enabled the wake word?  

That omission was intentional.  With the new audio code, we can actually start the wake word while playing a song, and it will still pick it up.

The only caveat is that it takes awhile to hit the VAD timeout (dont think thats configurable) so it can take 5-10 seconds after you finish speaking to process, but it does work.

Or perhaps, Im mis-understanding you, lmk!

Reply:
Hm, that is a fair point. I hadn't thought of that.

Reply:
It might be a good idea to stop playing when wake word is detected, though, right?